src/integration-tests/bash/run.sh

```diff
@@ -2698,6 +2703,9 @@ function test_suite {
       # create domain6 in the default namespace with pvReclaimPolicy="Recycle", and verify that the PV is deleted once the domain and PVC are deleted
       test_create_domain_pv_reclaim_policy_recycle domain6
 
+      # create domain7 in the default namespace with weblogicDomainStorageType="NFS", and verify it
+      test_domain_creation domain7
```

Comment:
What does this test do?  I see that it creates a domain with the "NFS" storage type, but does that NFS share exist.  If not, what have we proved or verified?

Reply:
Configure NFS as a persistent volume and test the domain creation.
Test cases use the local host's NFS server. $PV_ROOT/acceptance_test_pv/$DOMAIN_STORAGE_DIR is the shared nfspath in domain7

Reply:
Does this mean that we could no longer run the integration tests locally?  We need to either create the NFS share from within the test infrastructure or document the pre-requisites and allow the person running the tests to inject details about NFS shares to use.

Reply:
Yes. Our initial goal was to configure a dedicated remote NFS server, and test cases could only run on JENKINS machines.  Since we don't have any extra NFS shares. We have to use the local NFS server running on the JENKINS machine.